qiskit_addon_sqd/fermion.py

```diff
@@ -159,6 +159,9 @@ def solve_fermion(
         amplitudes=np.array(sci_vec), ci_strs_a=sci_vec._strs[0], ci_strs_b=sci_vec._strs[1]
     )
 
+    # Flip the occupancies to match the order of the bitstrings
+    avg_occupancy = list(np.flip(avg_occupancy))
```

Comment:
I think it's fine to keep it as a Numpy array? I think we can compute it directly in line 148 like this
```python
avg_occupancy = np.concatenate([np.diagonal(dm1[0]), np.diagonal(dm1[1])])[::-1]

Reply:
Maybe do the `concatenate` in line 148, and then do the reversal in the return statement, with a comment

Reply:
Fine by me, I'll just update the type hint. It's been a length-2 list containing 2 np.arrays

Reply:
Should we keep it a length-2 Python sequence or make it a 2D np array?

Reply:
(dice solver is a length-2 tuple lol ... we can make them more consistent if we decide here)

Reply:
Ohh yeah I was imagining just returning a single 1-d Numpy array

Reply:
I think we should return whatever is appropriate to pass directly to `recover_configurations`. That would mean a 1-D Numpy array, right?

Reply:
Agreed, done